client/flutter/lib/pages/onboarding/onboarding_page.dart

```diff
@@ -49,11 +56,20 @@ class _OnboardingPageState extends State<OnboardingPage> {
   }
 
   Future<void> _toNextPage() async {
-    await _pageController.nextPage(duration: _animationDuration, curve: _animationCurve);
+    await _pageController.nextPage(
+      duration: _animationDuration,
+      curve: _animationCurve,
+    );
   }
 
-  void _onDone() {
-    // Return true because onboarding is completed
-    Navigator.of(context).pop(true);
+  void _onDone() async {
+    await UserPreferences().setOnboardingCompleted(true);
+    await UserPreferences().setAnalyticsEnabled(true);
+    await Navigator.pushReplacement(
+      context,
+      MaterialPageRoute(
+        builder: (BuildContext context) => HomePage(widget.analytics),
```

Comment:
nitpick (consistency): only `c` or `context` is fine, this is what I see used throughout the app. Specifying type `BuildContext` is not needed.